gradle/libs.versions.toml

```diff
@@ -3,14 +3,14 @@ androidGradle = "7.4.2"
 androidxActivity = "1.5.0"
 androidxAppCompat = "1.3.1"
 androidxCompose = "1.2.0"
-androidxComposeCompiler = "1.4.4"
+androidxComposeCompiler = "1.5.2"
 androidxCore = "1.7.0"
 androidxFragment = "1.5.0"
 androidxLifecycle = "2.5.0"
 androidxWork = "2.7.0"
-anvil = "2.4.4"
+anvil = "2.4.7"
 dagger = "2.45"
-kotlin = "1.8.10"
+kotlin = "1.9.0"
```

Comment:
Are the dependency updates required for the fix to work? If not I'd rather leave them out

Reply:
Should be fine, dropped the commit.